qiime2/core/testing/format.py

```diff
@@ -98,3 +98,19 @@ def single_ints_path_maker(self, num):
             return 'nested/file%d.txt' % num
         else:
             return 'file%d.txt' % num
+
+
+class UnimportableFormat(TextFileFormat):
+    """
+    Useless format which maps to nothing, used for testing the plugin
+    manager.
+
+    """
+
+
+class UnimportableDirectoryFormat(model.DirectoryFormat):
```

Comment:
Can you make this a single-file directory format using the `model.SingleFileDirectoryFormat` factory? There's examples of that in this file, e.g. [this line](https://github.com/qiime2/qiime2/pull/275/files#diff-e363c313e0bcaae16f7f16b373d2ea2bL69)